index.html

```diff
@@ -322,50 +322,51 @@ <h2>The <dfn>PerformanceEntry</dfn> interface</h2>
         [Default] object toJSON();
       };</pre>
     <dl>
-      <dt><dfn>id</dfn></dt>
-      <dd>
-        This attribute MUST return the value of <a>this</a>'s <a>id</a>.
-      </dd>
       <dt><dfn>name</dfn></dt>
       <dd>
-        This attribute MUST return an identifier for this
-        <a>PerformanceEntry</a> object. This identifier does not have to be
-        unique.
+        This attribute must return the value it is initialized to. It represents an identifier for
+        this <a>PerformanceEntry</a> object. This identifier does not have to be unique.
       </dd>
       <dt><dfn>entryType</dfn></dt>
       <dd>
-        This attribute MUST return the type of the interface represented by
-        this <a>PerformanceEntry</a> object.
+        This attribute must return the value it is initialized to.
+
         <p class="note">All `entryType` values are defined in the
         relevant<a href=
         "https://w3c.github.io/timing-entrytypes-registry/#registry">registry</a>.
         Examples include: <code>"mark"</code> and <code>"measure"</code>
         [[USER-TIMING-2]], <code>"navigation"</code> [[NAVIGATION-TIMING-2]],
-        <code>"resource"</code> [[RESOURCE-TIMING-2]],
-        <!-- TODO: add long task spec reference -->
-         and <code>"longtask"</code>.</p>
+        and <code>"resource"</code> [[RESOURCE-TIMING-2]].</p>
       </dd>
       <dt><dfn>startTime</dfn></dt>
-      <dd>This attribute MUST return the time value of the first recorded
-      timestamp of this performance metric. If the startTime concept doesn't
-      apply, a performance metric may choose to return a `startTime` of
-      <code>0</code>.</dd>
+      <dd>This attribute must return the value it is initialized to. It represents the time value of
+      the first recorded timestamp of this performance metric. Its initial value is <code>0</code>.</dd>
```

Comment:
Can this have an "initial value", but also be "initialized" to something different?